src/js/factual_background.js

```diff
@@ -26,16 +28,25 @@ class TabInfo {
     this.updateBrowserAction();
   }
 
-  setLoading() {
-    this.status = 'loading';
+  setLoading(url) {
+    this.previousUrl = this.currentUrl;
+    this.currentUrl = url;
+  }
+
+  getCurrentUrl() {
```

Comment:
Ta metoda wydaje sie zbedna

Reply:
Dlaczego zbędna? Używam jej później, żeby zobaczyć czy url zakładki się zmienił: `getCurrentUrl() != getPreviousUrl()`

Reply:
Chodziło mi bardziej o to, że można się po prostu odwołać do zmiennej `currentUrl`, bo i tak jest publiczna. To samo w pozostałych przypadkach. To jest pierdółka, może zostać i tak ;)

Reply:
Myślałem, że to dobra praktyka, żeby robić gettery/settery. :) Usunę, będzie czystszy kod.